dev_config.ini

```diff
@@ -1,3 +1,4 @@
 [thangs]
 url = https://development.thangs.com/
 event_url = https://dev-events.thangs.com/
+fp_url = https://dev-fp.thangs.com/fp_m
```

Comment:
Do we not have a staging instance of this service right now?  I don't see an update to the staging configuration file

Reply:
We do not have a staging service